hail/python/hailtop/hailctl/dataproc/modify.py

```diff
@@ -57,10 +56,8 @@ def main(args, pass_through_args):
 
     wheel = None
     if args.update_hail_version:
-        assert pkg_resources.resource_exists('hailtop.hailctl', "deploy.yaml")
-        updated_wheel = yaml.safe_load(
-            pkg_resources.resource_stream('hailtop.hailctl', "deploy.yaml"))['dataproc']['wheel']
-        wheel = updated_wheel
+        deploy_metadata = get_deploy_metadata()
```

Comment:
Moved this to a separate function so that it's easier to mock.

Reply:
👍